src/utils/helpers.py

```diff
@@ -0,0 +1,33 @@
+import requests
+from PIL import Image
+from io import BytesIO
+from collections import Counter
+
+BASE_URL = "https://dxbhsrqyrr690.cloudfront.net/sidearm.nextgen.sites/cornellbigred.com"
+
+def get_dominant_color(relative_path, white_threshold=200, black_threshold=50):
+    image_url = f"{BASE_URL}{relative_path}"
+    
+    response = requests.get(image_url)
+    image = Image.open(BytesIO(response.content)).convert("RGBA")
+    
+    image = image.resize((50, 50))
+    image = image.quantize(colors=5).convert("RGBA")
```

Comment:
Did quantizing to 5 colors work the best? I'm wondering if quantizing to more colors might work better but this seems fine as it is right now